### PR TITLE
Do not allow single player to sleep in daytime

### PR DESCRIPTION
**Pull request by trigg**
Check that it is the night before anything else. As it currently stands players can skip to the next day if they are alone on a server.

### CODE REVIEW

**Comment by luacs1998** [comment]
Is this standard behaviour without FE?

**Comment by trigg** [comment]
The normal MC rule is "You cannot sleep at night!" Message being thrown (and still is) and the time remains the same and the player remains out of bed. The bug here is that time is pushed forward regardless.

File: src/main/java/com/forgeessentials/util/MiscEventHandler.java [hunk 1]
```diff
@@ -53,7 +57,6 @@ public void playerSleepInBedEvent(PlayerSleepInBedEvent e)
             OutputHandler.felog.finer("Players sleeping: " + percent + "%");
             if (percent > 50)
```
Comment (olee) [inline src/main/java/com/forgeessentials/util/MiscEventHandler.java]: Shouldn't we make this value configurable at the same time?

**Comment by luacs1998** [comment]
I will fix this manually myself, seeing as travis can't pass a judgement and I don't want to merge it without knowing how it works.

Might refactor the MajoritySleep thing too.

**Comment by luacs1998** [comment]
Closed in 98ec78815d3c814a00c287fabb305e3c63a96cb2

**Pull request closed without merging**